### DISCUSSION TITLE
Disconnect

### DISCUSSION THREAD

**Discussion by FabianGuimaraes** [sandboxie-plus/Sandboxie#3853] [Q&A Feedback]
![imagem_2024-04-26_210953333](https://github.com/sandboxie-plus/Sandboxie/assets/168240837/26ec21ee-c72e-459c-bae8-ffecb5ad0ba1)

Someone can help me?
I only trying to open 2 games (CABAL ONLINE) for go to DUNGEON, how can i use this, because befor 10 minutes the program closed...

**Comment by bastik-1001** [comment]
Since the processes are crashing, there is a compatibly issue. You could take a look at [ways to figure out what's going wrong](https://github.com/sandboxie-plus/Sandboxie/blob/master/CONTRIBUTING.md#how-to-help-with-sandboxie-issues).

**Reply by FabianGuimaraes** [reply]
I'm trying to play 2 open game ( cabal online ) i believe is ( XIGNCODE3 100158 ) is closing how can i solve this? to no detect 2 games open.

**Reply by FabianGuimaraes** [reply]
i really want thiss can u help me?

**Reply by bastik-1001** [reply]
I am not in a position to test this. You could try a compartment box, which can only be tested for a limited time (10 minutes) and this might be too short. I believe, that David still gives out a limited certificate for evaluating if Sandboxie is going to be useful in that instance.

**Reply by FabianGuimaraes** [reply]
ty <3

**Reply by camsay1996** [reply]
same having the same error while using sandbox for cabal. xigncod3 close it
